src/composables/transactionList.ts

```diff
@@ -0,0 +1,338 @@
+import { useStore } from 'vuex';
+import { orderBy, uniqBy } from 'lodash-es';
+import {
+  computed,
+  onMounted,
+  ref,
+  watch,
+} from 'vue';
+import { Encoded, Tag } from '@aeternity/aepp-sdk';
+
+import type {
+  ITransaction,
+  ITransactionsState,
+  ITransactionStateList,
+} from '../types';
+import { useMiddleware } from './middleware';
+import {
+  AEX9_TRANSFER_EVENT,
+  fetchJson, getLocalStorageItem, setLocalStorageItem,
+  TX_DIRECTION,
+} from '../popup/utils';
+import { useAccounts } from './accounts';
+import JsonBig from '../lib/json-big';
+import { useAeSdk } from './aeSdk';
+
+const LOCAL_STORAGE_KEY = 'transactions_state';
+
+const isInitalized = ref(false);
```

Comment:
Typo
I strongly suggest to install some kind of "Code Spell Checker" plugin to your IDE.